### ISSUE TITLE
Oddness with requireSpacesInsideParentheses, quotes, and multiple args

### ISSUE THREAD

**Issue by texel** [jscs-dev/node-jscs#1916]
Per https://github.com/rwaldron/idiomatic.js/#spacing, I'm trying to support this:

```javascript
// Function call with multiple arguments
foo( "bar", baz );

// Single argument string literal, no space
foo("bar");
```

With the current rules, I can use `"except": ['"']` but that will result in the following fixes:

```javascript
foo("bar", baz) // becomes
foo("bar", baz )

foo(baz, "bar") // becomes
foo( baz, "bar")
```

It seems like we might need explicit exceptions for function arguments. I'm fine with digging in and attempting a fix, but I wanted to get the maintainers' opinions first on how best to tackle this.

**Comment by markelog** [comment]
How about we make it as separate rule? Two rules even, since one would require to have spaces -

```js
foo( baz, "bar" )
```
and the other would disallow it - 
```js
foo("bar")
```

Does it sound doable?

**Comment by texel** [comment]
Sure, that sounds reasonable.

**Comment by markelog** [comment]
Would you like to take a stab at it?

**Comment by texel** [comment]
Sorry for the delay... yeah I'd like to.

**Comment by hzoo** [comment]
I'm putting an [`orphaned`](https://github.com/jscs-dev/node-jscs/issues?q=label%3Aorphaned+is%3Aclosed) label to note this issue isn't planned to be worked on, but might be a good contribution as a custom rule (not in core). We can all check back on this list for future rule ideas/options.

We released [3.0](https://github.com/jscs-dev/node-jscs/releases/tag/v3.0.0) recently! We also wrote a [post](https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.oot3wvgpu) about how 3.0 will be our last major release (since we are deprecating the project) and that we are [merging with ESLint](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint)!

Thus we won't be working on any new features moving forward which includes new rules/options. We will continue to fix bugs and work on CST/better autofixing for current rules in the upcoming months both to help JSCS users and to gain experience that we can hopefully use in ESLint. But otherwise we both teams will be working on getting [ESLint feature parity with JSCS](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) and just a nice transition for our users.

I would encourage anyone who wants the rule to try writing it themselves: I would either write a custom rule or submit an issue to ESLint and if accepted a PR. Otherwise a custom rule that won't be in core repo itself.

JSCS rules are also very similar to ESLint rules: here are some resources to get started: 

- [Project-specific Lint Rules with ESLint](https://gist.github.com/jareware/7179093)
- [Creating Custom Rules for ESLint](https://insideops.wordpress.com/2015/12/08/creating-custom-rules-for-eslint)
- [Maintaining JavaScript Code Quality with ESLint ](https://www.paypal-engineering.com/2014/12/12/maintaining-javascript-code-quality-with-eslint/)
